### ISSUE TITLE
Can't emerge elementary-icon-theme-4.2.0

### ISSUE THREAD

**Issue by squat80** [pimvullers/elementary#181]
Hi i'm trying to emerge pantheon-shell on a brand new system
i pretty much followed the instruction from https://wiki.gentoo.org/wiki/Pantheon
(added the few uses/accept_keywords that portage requested)

but i get an error when emerging elementary-icon-theme-4.2.0
it instantly fails with
```
>>> Preparing source in /var/tmp/portage/x11-themes/elementary-icon-theme-4.2.0/work/elementary-icon-theme-4.2.0 ...
 * Unable to locate CMakeLists.txt under:
 * "/var/tmp/portage/x11-themes/elementary-icon-theme-4.2.0/work/elementary-icon-theme-4.2.0/CMakeLists.txt"
 * Consider not inheriting the cmake eclass.
 * ERROR: x11-themes/elementary-icon-theme-4.2.0::elementary failed (prepare phase):
 *   FATAL: Unable to find CMakeLists.txt
```
i don't see the CMakeLists.txt anywhere in the source package nor in the ebuilds folder

any ideas ?
/var/tmp/portage/x11-themes/elementary-icon-theme-4.2.0/temp/environment in attachement
[environment.log](https://github.com/pimvullers/elementary/files/1327502/environment.log)

thanks for your help
David

**Comment by terwey** [comment]
I had this too and fixed it in #183

**Comment by pimvullers** [comment]
The fix of terwey has been delivered.
